index.html

```diff
@@ -2,31 +2,30 @@
 layout: default
 ---
 
-<h1>{{ site.title }}</h1>
+<section>
+    <h1>{{ site.title }}</h1>
 
-<p>Please enter atendees below (each in a separate line):</p>
-<textarea></textarea>
+    <p>Please enter attendees below (each in a separate line):</p>
+    <textarea id="attendees"></textarea>
 
-<button class="split">Split in groups</button>
+    <button id="split" class="split">Split in groups</button>
+</section>
 
-<table>
+<section>
+    <h2>Results</h2>
+    <table>
 
-	<thead>
-		<tr>
-			<th>Group A</th>
-			<th>Group B</th>
-			<th>Group C</th>
-			<th>Group D</th>
-		</tr>
-	</thead>
+    <thead>
+        <tr>
+            <th>Group A</th>
+            <th>Group B</th>
+            <th>Group C</th>
+            <th>Group D</th>
+       </tr>
+    </thead>
 
```

Comment:
There are tabs there instead of spaces I think. I could be wrong about reading the bold green diff.